### PR TITLE
Add back GC_EVENTS to ES and display GC Evengts to yarn and standalone dashboards

### PR DESCRIPTION
**Pull request by ashangit**
It represents around 30 % more events on ES

### CODE REVIEW

File: readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java [hunk 1]
```diff
@@ -61,7 +59,7 @@
 
     ElasticSearchReader(GarmadonReader.Builder builderReader, BulkProcessor bulkProcessorMain, String esIndexPrefix, PrometheusHttpConsumerMetrics prometheusHttpConsumerMetrics) {
         this.reader = builderReader
-                .intercept(not(hasType(GarmadonSerialization.TypeMarker.GC_EVENT)), this::writeToES)
+                .intercept(GarmadonMessageFilter.ANY.INSTANCE, this::writeToES)
```
Comment (gray-matter) [inline readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: Unrelated change?

Comment (ashangit) [reply readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: This is completely related.
Without that we still filter GC event from readers so never sending GC events to ES ;)

Comment (gray-matter) [reply readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java]: Indeed. My review skills are not that good this year :wink:
